### PR TITLE
OpenApi - attributesManager methods

### PR DESCRIPTION
**Pull request by Vojtech-Sassmann**
* Added new descriptions of methods to the openApi.
* Created new schemas for optional id parameters.

* getAttributeDefinitionWithRights
* removeAttribute
* updateAttributeDefinition
* getAttributeRights
* setAttributeRights

### CODE REVIEW

File: perun-openapi/openapi.yml [hunk 1]
```diff
@@ -202,6 +202,27 @@ components:
...
+      enum:
+        - "WRITE"
+        - "WRITE_VO"
+        - "WRITE_PUBLIC"
+        - "READ"
+        - "READ_VO"
+        - "READ_PUBLIC"
+
+    AttributeRights:
+      allOf:
```
Comment (martin-kuba) [inline perun-openapi/openapi.yml]: allOf is not needed when the type does not inherit from another type

File: perun-openapi/openapi.yml [hunk 2]
```diff
@@ -2892,7 +2994,28 @@ paths:
...
 
-#TODO getAttributesDefinitionWithRights
+  /urlinjsonout/attributesManager/getAttributesDefinitionWithRights:
+    get:
+      tags:
+        - AttributesManager
+      operationId: getAttributesDefinitionWithRights
+      summary: Returns all AttributeDefinitions for every entity and possible
+        combination of entities with rights. Only attribute definition of 
+        attributes user can read (or write) you will get.
```
Comment (martin-kuba) [inline perun-openapi/openapi.yml]: The javadoc also contains the following, please add it too:
```
Combination of entities is based on provided parameters, which are optional (at least one must be present).
```

File: perun-openapi/openapi.yml (hunk 1 above)
Comment (Vojtech-Sassmann) [reply perun-openapi/openapi.yml]: Fixed.

File: perun-openapi/openapi.yml (hunk 2 above)
Comment (Vojtech-Sassmann) [reply perun-openapi/openapi.yml]: Summary updated.

**Pull request merged**